packages/frontend/src/components/ProjectConnection/DbtForms/GithubForm.tsx

```diff
@@ -48,30 +50,52 @@ const GithubLoginForm: FC<{ disabled: boolean }> = ({ disabled }) => {
             register('dbt.installation_id', { value: config?.installationId });
         }
     }, [config?.installationId, register]);
+    const { showToastSuccess } = useToaster();
 
-    return (
-        <>
-            {isValidGithubInstallation ? (
-                <>
-                    {' '}
+    const DropdownComponentOverride = useCallback(
+        ({ children, ...props }: { children: ReactNode }) => (
+            <Stack w="100%" spacing={0}>
+                <ScrollArea {...props}>{children}</ScrollArea>
+
+                <Tooltip
+                    withinPortal
+                    position="left"
+                    width={300}
+                    multiline
+                    label="Click here to open your Github installation page to add more repositories."
+                >
                     <Text
+                        color="dimmed"
+                        size="xs"
+                        px="sm"
+                        p="xxs"
                         sx={(theme) => ({
-                            transition: 'color 0.3s ease',
-                            animation: 'fadeGreen 3s',
-                            '@keyframes fadeGreen': {
-                                '0%': { color: theme.colors.green[4] },
-                                '100%': { color: 'inherit' },
+                            cursor: 'pointer',
+                            borderTop: `1px solid ${theme.colors.gray[2]}`,
+                            '&:hover': {
+                                backgroundColor: theme.colors.gray[1],
                             },
                         })}
+                        onClick={() =>
+                            window.open(
+                                `https://github.com/settings/installations/${config?.installationId}`,
+                                '_blank',
+                            )
+                        }
                     >
-                        You are connected to GitHub.{' '}
-                        <Anchor
-                            href={'/generalSettings/integrations'}
-                            target="_blank"
-                        >
-                            Click here to use another account
-                        </Anchor>{' '}
+                        Don't see your repository?{' '}
+                        <Anchor>Configure here</Anchor>
                     </Text>
+                </Tooltip>
+            </Stack>
+        ),
+        [config?.installationId],
+    );
```

Comment:
[non-blocker] could this be extracted on top level of the file? seems like it just gets one variable from the closure scope, can be a prop